From mcarlson@broadcom.com  Thu Jun 23 22:01:20 2011
From: Matt Carlson <mcarlson@broadcom.com>
Date: Wed, 22 Jun 2011 18:56:09 -0700
Subject: [PATCH 002/194] tg3: Add tg3_compat.h
Patch-mainline: Never, used for backports only
References: bnc#697783, FATE#311457

This patch adds tg3_compat.h, to be filled in as needed.
Acked-by: Jeff Mahoney <jeffm@suse.com>
---
 drivers/net/tg3.h        |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)
 create mode 100644 drivers/net/tg3_compat.h

diff --git a/drivers/net/tg3.h b/drivers/net/tg3.h
index c56f5ac..6a64c8c 100644
--- a/drivers/net/tg3.h
+++ b/drivers/net/tg3.h
@@ -10,6 +10,8 @@
 #ifndef _T3_H
 #define _T3_H
 
+#include "tg3_compat.h"
+
 #define TG3_64BIT_REG_HIGH		0x00UL
 #define TG3_64BIT_REG_LOW		0x04UL
 
diff --git a/drivers/net/tg3_compat.h b/drivers/net/tg3_compat.h
new file mode 100644
index 0000000..e69de29
-- 
1.7.3.4

